### PR TITLE
add support for ovs offload

### PR DESCRIPTION
**Pull request by Mmduh-483**

### CODE REVIEW

**Comment by Mmduh-483** [comment]
@moshe010

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,7 @@ module github.com/vmware-tanzu/antrea
 go 1.13
 
 require (
+	github.com/Mellanox/sriovnet v0.0.0-20191103093832-a16b5a5bd280
```
Comment (moshe010) [inline go.mod]: please use tag v1.0.1

File: go.mod [hunk 2]
```diff
@@ -26,7 +27,7 @@ require (
 	github.com/j-keck/arping v1.0.0
 	github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd
 	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
-	github.com/satori/go.uuid v1.2.0
+	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b // indirect
```
Comment (moshe010) [inline go.mod]: run go mod tidy will remove this

**Review by moshe010 — CHANGES_REQUESTED** [review]
how the validate interface works for you?
https://github.com/Mmduh-483/antrea/blob/master/pkg/agent/cniserver/pod_configuration.go#L191

**Comment by moshe010** [comment]
also see if we can add unit-test to https://github.com/moshe010/antrea/blob/master/pkg/agent/cniserver/server_test.go

**Pull request closed without merging**